tests/test_venue_request_v2.py

```diff
@@ -585,21 +586,21 @@ def test_venue_recruitment(self, client, test_client, selenium, request_page, ve
 
         invalid_accept_url = re.search('https://.*\n', messages[0]['content']['text']).group(0).replace('https://openreview.net', 'http://localhost:3030').replace('&amp;', '&')[:-1].replace('user=reviewer_candidate2_v2%40mail.com', 'user=reviewer_candidate2_v1%40mail.com')
         print(invalid_accept_url)
-        helpers.respond_invitation(selenium, request_page, invalid_accept_url, accept=True)
+        helpers.respond_invitation(selenium, request_page, invalid_accept_url, accept=True, allow_accept_with_reduced_load=True)
```

Comment:
I would prefer to add a new test instead of editing an existing one. 

When you pass `allow_accept_with_reduced_load=True` don't you need to pass the quota value?

I think we don't need to add a new parameter, I think the function respond_invitation should handle this automatically: 

if accept is True and quota is not None then we should try to find the button to accept with reduced load. If the button is not found then we should decline the invitation and accept the invitation with reduced load.

Reply:
Ah okay, I see what you mean. I didn't see `quota` being passed in anywhere to `respond_invitation()` so I didn't fully understand what it was, but I understand now. And basically all this test was doing was checking if the Accept with Reduced Load button gets displayed. I'll update the tests.

Reply:
`quota` is the reduced_load index that you want to select for that reviewer that is accepting the invitation. Previously users could only select the reduced load rejecting the invitation first and then accepting the invitation with a quota. That is what this code does. Now that we want to support the acceptance with the reduce load without having to reject first, this function should check if this option is available (check with selenium if the button exists) and then accept the invitation selecting the reduce load, if the button is not found then you do the existing behavior (reject first and then accept with reduce load)